test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```

Comment:
- [Optional] Consider removing this file. You don't need the file. It was just used in the hello microverse project to demonstrate how a file can be ignored by git.

Reply:
Done. Thanks.